suite-native/receive/src/components/UnverifiedAddressDevice.tsx

```diff
@@ -85,18 +125,21 @@ export const UnverifiedAddressDevice = ({
 
     return (
         <VStack spacing="medium" alignItems="center">
-            <Box style={applyStyle(deviceFrameStyle)}>
+            <Animated.View style={deviceFrameStyle}>
                 <PanGestureHandler onGestureEvent={handlePaginationPanGesture}>
                     <Animated.View>
-                        <VStack
-                            spacing="large"
-                            style={applyStyle(deviceScreenStyle, { isPaginationEnabled })}
-                        >
+                        <Animated.View style={deviceScreenStyle}>
                             <DeviceScreenContent
                                 address={address}
-                                isAddressRevealed={isAddressRevealed}
+                                isAddressRevealed={isAddressRevealed && isConfirmOnTrezorReady}
```

Comment:
Please just use `isConfirmOnTrezorReady`

Reply:
changed in b440128